documentation/config_file_puppetserver.md

```diff
@@ -90,6 +99,12 @@ http-client: {
 profiler: {
     enabled: true
 }
-~~~
+
+# Settings related to static catalogs.
+versioned-code: {
+    code-id-command: /opt/puppetlabs/server/apps/puppetserver/code-id-command_script.sh
+    code-content-command: /opt/puppetlabs/server/apps/puppetserver/code-content-command_script.sh
+}
+```
```

Comment:
We could, though that would add a lot of Puppet code to all the configuration docs that might need to be updated if the module changes how it works. Since we can't assume everyone will use a module (or our module) to manage Server's configuration files, the individual configuration file pages should probably stick to the basics.

We could add a section to the configuration page that notes the module and provides a generic example that could apply to any of the Server configuration files, but that's out of this PR's scope.